test/Atmos/EDMF/ekman_layer.jl

```diff
@@ -0,0 +1,271 @@
+
+include(joinpath(clima_dir, "experiments", "AtmosLES", "ekman_layer_model.jl"))
+include(joinpath("helper_funcs", "diagnostics_configuration.jl"))
+include("edmf_model.jl")
+include("edmf_kernels.jl")
+
+CLIMAParameters.Planet.T_surf_ref(::EarthParameterSet) = 290.0
+function set_clima_parameters(filename)
+    eval(:(include($filename)))
+end
+
+"""
+    init_state_prognostic!(
+            turbconv::EDMF{FT},
+            m::AtmosModel{FT},
+            state::Vars,
+            aux::Vars,
+            localgeo,
+            t::Real,
+        ) where {FT}
+
+Initialize EDMF state variables if turbconv=EDMF(...) is selected.
+This method is only called at `t=0`.
+"""
+function init_state_prognostic!(
+    turbconv::EDMF{FT},
+    m::AtmosModel{FT},
+    state::Vars,
+    aux::Vars,
+    localgeo,
+    t::Real,
+) where {FT}
+    # Aliases:
+    gm = state
+    en = state.turbconv.environment
+    up = state.turbconv.updraft
+    N_up = n_updrafts(turbconv)
+    z = altitude(m, aux)
+
+    param_set = parameter_set(m)
+    ts = new_thermo_state(m, state, aux)
+    θ_liq = liquid_ice_pottemp(ts)
+
+    a_min = turbconv.subdomains.a_min
+    @unroll_map(N_up) do i
+        up[i].ρa = gm.ρ * a_min
+        up[i].ρaw = gm.ρu[3] * a_min
+        up[i].ρaθ_liq = gm.ρ * a_min * θ_liq
+        up[i].ρaq_tot = FT(0)
+    end
+
+    en.ρatke =
+        z > FT(250) ? FT(0) :
+        gm.ρ *
+        FT(0.4) *
+        FT(1 - z / 250.0) *
+        FT(1 - z / 250.0) *
+        FT(1 - z / 250.0)
+    en.ρaθ_liq_cv = FT(0)
+    en.ρaq_tot_cv = FT(0)
+    en.ρaθ_liq_q_tot_cv = FT(0)
+    return nothing
+end;
+
+function main(::Type{FT}, cl_args) where {FT}
+    # Change boolean to control vertical discretization type
+    finite_volume = false
+
+    # Choice of compressibility and CFL
+    # compressibility = Compressible()
+    compressibility = Anelastic1D()
+    str_comp = compressibility == Compressible() ? "COMPRESS" : "ANELASTIC"
+
+    # Choice of SGS model
+    turbconv = NoTurbConv()
+    # N_updrafts = 1
+    # N_quad = 3
+    # turbconv = EDMF(FT, N_updrafts, N_quad, param_set)
+
+    C_smag_ = C_smag(param_set)
+    turbulence = ConstantKinematicViscosity(FT(0.1))
+    # turbulence = SmagorinskyLilly{FT}(C_smag_)
+
+    # Prescribe domain parameters
+    zmax = FT(400)
+    # Simulation time
+    t0 = FT(0)
+    timeend = FT(3600 * 2) # Change to 7h for low-level jet
+    CFLmax = compressibility == Compressible() ? FT(1) : FT(100)
+
+    config_type = SingleStackConfigType
+    ode_solver_type = ClimateMachine.ExplicitSolverType(
+        solver_method = LSRK144NiegemannDiehlBusch,
+    )
+
+    if finite_volume
+        N = (1, 0)
+        nelem_vert = 80
+        ref_state = HydrostaticState(
+            DryAdiabaticProfile{FT}(param_set),
+            ;
+            subtract_off = false,
+        )
+        output_prefix = string("EL_", str_comp, "_FVM")
+        fv_reconstruction = FVLinear()
+    else
+        N = 4
```

Comment:
do you want to change to 
N=1
nelem_vert = 80
here?

Reply:
not at the moment, I want to leave the default for #2116